### ISSUE TITLE
[Request] Add Advanced Blocking Card

### ISSUE THREAD

**Issue by MicroObsidianBlock** [GlodBlock/AE2FluidCraft-Rework#116]
Can you port the advanced blocking card from GTNH for interfaces. I am posting here because I think to be able to block the fluids, it's only possible to ask here rather than in PAE2's.

[https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/pull/308]

I will cite this in PAE2's issues.

**Comment by YiRanMushroom** [comment]
You are right, the assembly line automation is really a pain at 1.12.2, espetially when it now need to be ordered

**Comment by YiRanMushroom** [comment]
I checked the source code of the card and found out that the advanced blocking card is actually hard to code. Instead, I think try to do a lock feature of the interface could be easier, like that in 1.19.2, lock craft until primary output is inserted. The thing for advanced blocking card is hard because the storage bus would recognize the network that it is facing, rather than a normal inventory. So I think add the lock feature would work as you want. Also it's hard for the mod to do it (at least I don't know how to do without directly editing PAE2). So maybe you could go to there to show the problem.

**Comment by YiRanMushroom** [comment]
Also to detect the primary output is hard, but I don't think we need to do that, we only need to detect if there is items being imported to the interface and that is much easier.

**Comment by MicroObsidianBlock** [comment]
Yes, that is an option, and I think is better than the advanced blocking card. I don't know if it is hard to check the primary output, but to check if there is item going in should not be that hard

**Comment by GlodBlock** [comment]
it seems that ae2 itself already supports this feature, interface blocking mode will check the me network inventory instead of interface itself.

**Comment by CactusCool** [comment]
> it seems that ae2 itself already supports this feature, interface blocking mode will check the me network inventory instead of interface itself.

yea but sadly it only works in regular interfaces and not with duals. Even if the same blocking logic the regular interfaces use (only items) was copied to the dual interface it would still be fine as the discretizer should expose fluid as items in the network allowing it to be blocked.
